firmware/hw_layer/hardware.cpp

```diff
@@ -390,9 +390,9 @@ void applyNewHardwareSettings(DECLARE_ENGINE_PARAMETER_SIGNATURE) {
 	}
 #endif
 
-#if EFI_VEHICLE_SPEED && ! EFI_UNIT_TEST
+/* #if EFI_VEHICLE_SPEED && ! EFI_UNIT_TEST
```

Comment:
we can remove these fully in the next PR, don't worry about it for now to keep this PR clean.